db/schema.rb

```diff
@@ -10,11 +10,26 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
+ActiveRecord::Schema.define(version: 20170918000238) do
+=======
 ActiveRecord::Schema.define(version: 20170915132625) do
+>>>>>>> master
 
```

Comment:
Looks like this merge conflict got staged. Let's remove it before we merge to master. Thanks!

Reply:
Strange! I know I resolved this one!